app/pages/resource/info/ResourceInfo.js

```diff
@@ -22,6 +22,7 @@ function renderHeader(unit, resource) {
   const address = `${streetAddress}, ${zip} ${city}`;
   return (
     <header>
+      <Button className="back-link" bsSize="sm" href="/">{'<< Tilahaku'}</Button>
```

Comment:
- [ ] The link should use React-Router's Link. Otherwise it causes a full page reload.